README.md

```diff
@@ -195,8 +196,6 @@ java -jar target/migration-utils-6.0.0-SNAPSHOT-driver.jar --source-type=legacy
 
 † The `fedora3model` namespace is not a published namespace. It is a representation of the fcrepo 3 namespace `info:fedora/fedora-system:def/model`.
 
-‡ Not yet implemented
```

Comment:
It looks like this was for the tables above as this dagger is present in them. Definitely a bit awkward with the note below but I think this should be added back. 

It's probably worth it to go over the README at some point as some of these mappings are quite old and it would be good to verify them.

Reply:
Agreed especially as the print out of the `--help` is missing arguments, like the `-A, --atomic-resources` one.

Reply:
Actually, this PR should be rolled in here or something https://github.com/fcrepo-exts/migration-utils/pull/197